easyaudit/utils.py

```diff
@@ -60,3 +63,19 @@ def model_delta(old_model, new_model):
         delta = None
 
     return delta
+
+
+def get_m2m_field_name(model, instance):
+    """
+    Finds M2M field name on instance
+    Called from m2m_changed signal
+    :param model: m2m_changed signal model.
+    :type model: Model
+    :param instance:m2m_changed signal instance.
+    :type new: Model
+    :return: ManyToManyField name of instance related to model.
+    :rtype: str
+    """
+    for x in model._meta.related_objects:
```

Comment:
Might be more readable as  `next((...), None)`? or as a `for: else` loop construct,  since although it is expected that the function will find it's match, it appears that it might not. That said, this is quite minor, so I wouldn't worry about it.